src/main/java/io/cdap/plugin/dlp/DLPTransformPluginConfig.java

```diff
@@ -45,6 +47,7 @@
   public static final String CUSTOM_TEMPLATE_PATH_NAME = "customTemplatePath";
   public static final String TEMPLATE_ID_NAME = "templateId";
   public static final String CUSTOM_TEMPLATE_ENABLED_NAME = "customTemplateEnabled";
+  public static final String DLP_LOCATION = "dlpLocation";
```

Comment:
nit: dlpLocation -> location. 

DLP is redundant and 'location' is a well understood term in GCP.